extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -353,7 +354,7 @@ void ConsumeWindowsEventLog::onTrigger(const std::shared_ptr<core::ProcessContex
     context->yield();
     return;
   }
-  const utils::ScopeGuard guard_hEventResults([hEventResults]() { EvtClose(hEventResults); });
+  const gsl::finally guard_hEventResults([hEventResults]() { EvtClose(hEventResults); });
```

Comment:
same as [this](https://github.com/apache/nifi-minifi-cpp/pull/849#discussion_r460123115)